### PR TITLE
Revamp README.md

### PR DESCRIPTION
**Pull request by GuGuss**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -101,17 +110,19 @@ They are listed here just in case you get lost.
...
 
-4. Scale the backend app container (Python):
 
+Congratulations, you have completed the demo!
+
+You can now enjoy more Upsun capabilities like:
+* Scale your applications horizonally:
 - ```
-  upsun resources:set --count backend:3
+  upsun resources:set --count backend:3 --count frontend:3
```
Comment (pjcdawkins) [inline readme.md:119]:
```suggestion
  upsun resources:set --count backend:3,frontend:3
```

File: readme.md [hunk 2]
```diff
@@ -1,31 +1,37 @@
...
 
-2. Set the remote for the project, and first push:
+3. Create a new Upsun project:
 
 - ```
-  PROJECT_ID=$(upsun project:list --title "Upsun demo" --pipe)
+  upsun project:create
   ```
+
+4. Set the `upsun` Git remote:
```
Comment (fabpot) [inline readme.md:27]: I would remove this step and set `--set-remote` above (or even without adding the flag, the interactive mode asks the question, right now the default answer is N, but I suppose it should be Yes).

Comment (pjcdawkins) [reply readme.md:27]: https://github.com/platformsh/legacy-cli/pull/1325

File: readme.md [hunk 3]
```diff
@@ -1,31 +1,37 @@
...
-  upsun create --org upsun-testing --title "Upsun demo" --region "org.recreation.plat.farm" --plan flexible --default-branch main --no-set-remote -y
+  upsun organization:create
   ```
 
-2. Set the remote for the project, and first push:
+3. Create a new Upsun project:
 
 - ```
-  PROJECT_ID=$(upsun project:list --title "Upsun demo" --pipe)
+  upsun project:create
```
[the remark this answers is not in the document]
Comment (tylers-username) [reply readme.md:24]: I generally like this option, but for new users, would it be better to use the more descriptive `project:create` option to clarify the action being taken?

File: readme.md [hunk 4]
```diff
@@ -1,31 +1,37 @@
...
   ```
+
+4. Set the `upsun` Git remote:
+
 - ```
-  upsun project:set-remote $PROJECT_ID
+  upsun project:set-remote PROJECT_ID
```
[the remark this answers is not in the document]
Comment (tylers-username) [reply readme.md:30]: > https://github.com/platformsh/demo-project/pull/22#discussion_r1332907642
> I generally like this option, but for new users, would it be better to use the more descriptive project:create option to clarify the action being taken?

File: readme.md (hunk 2 above)
Comment (fabpot) [reply readme.md:27]: Thank you @pjcdawkins 
This step can definitely be removed now :)

**Pull request closed without merging**